### ISSUE TITLE
Unable to ignore jinja[spacing] rule on block scalar variable

### ISSUE THREAD

**Issue by setpill** [ansible/ansible-lint#4596]
##### Summary

Special case of #3826. For the `var-naming[no-role-prefix]` rule, the suggested solution works, but not for `jinja[spacing]`

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```console (paste below)
ansible-lint --version
ansible-lint 25.4.0 using ansible-core:2.18.5 ansible-compat:25.1.5 ruamel-yaml:0.18.10 ruamel-yaml-clib:None
```

- ansible installation method: pip
- ansible-lint installation method: pip

##### STEPS TO REPRODUCE

1. Create a block scalar variable with jinja2 template strings that somehow do not fulfill ansible-lint's rules.
2. Tell ansible-lint to ignore it.

`vars/main.yml`:

```yaml
---
somevar: >- # noqa: jinja[spacing] var-naming[no-role-prefix]
  {{- "some" }}
  {{- "stuff" }}
  {{- "split" }}
  {{- "over" }}
  {{- "lines" }}
  {{- "for" }}
  {{- "readability" }}
  {{- "with" }}
  {{- "whitespace" }}
  {{- "stripped" }}
```

##### Desired Behavior

```console
ansible-lint vars/main.yml
Passed: 0 failure(s), 0 warning(s) on 1 files. Last profile that met the validation criteria was 'production'.
```

##### Actual Behavior

```console
ansible-lint vars/main.yml
WARNING  Listing 1 violation(s) that are fatal
jinja[spacing]: Jinja2 spacing could be improved: {{- "some" }} {{- "stuff" }} {{- "split" }} {{- "over" }} {{- "lines" }} {{- "for" }} {{- "readability" }} {{- "with" }} {{- "whitespace" }} {{- "stripped" }} -> {{- "some" }}{{- "stuff" }}{{- "split" }}{{- "over" }}{{- "lines" }}{{- "for" }}{{- "readability" }}{{- "with" }}{{- "whitespace" }}{{- "stripped" }} (warning)
vars/main.yml:2:10 Jinja2 template rewrite recommendation: `{{- "some" }}{{- "stuff" }}{{- "split" }}{{- "over" }}{{- "lines" }}{{- "for" }}{{- "readability" }}{{- "with" }}{{- "whitespace" }}{{- "stripped" }}`.

Read documentation for instructions on how to ignore specific rule violations.

# Rule Violation Summary

  1 jinja profile:basic tags:formatting

Passed: 0 failure(s), 1 warning(s) on 1 files. Last profile that met the validation criteria was 'min'.
```
